libs/utils/utils.js

```diff
@@ -564,6 +547,9 @@ export const analyticsDecorateList = (li, idx) => {
   link.setAttribute('daa-ll', `${analyticsGetLabel(label)}-${idx + 1}`);
 };
 
+export const utf8ToB64 = (str) => window.btoa(unescape(encodeURIComponent(str)));
```

Comment:
nit: we should probably switch out the half deprecated methods "unescape" and "escape" here with something like: 
```
export const utf8ToB64 = (str) => window.btoa(decodeURI(encodeURIComponent(str)));
export const b64ToUtf8 = (str) => decodeURIComponent((window.atob(str)).replace('\\x', '%'));
```

Reply:
@vhargrave probably a good idea, but I don't want to risk breaking any current encodings.  Will take some testing so I'll have to get back to this at another time.